ajs-app.ts

```diff
@@ -2,7 +2,7 @@
 
 import './components/add-layers/add-layers.module';
 import './components/core';
-import './components/datasource-selector/datasource-selector.module';
+import './components/datasource-selector';
```

Comment:
This import can be removed, because datasource-selector is now imported from HsCore